### DISCUSSION TITLE
A collection of PIL and NumPy functions: drawerFunctions

### DISCUSSION THREAD

**Discussion by DeusAres** [python-pillow/Pillow#5943] [Show and tell]
## What is it?
It's a collection of functions and little scripts to manipulate images, written with PIL and Numpy (and openCV somewhere)

## Why?
I'm a lazy coder with bad memory, and I've found myself more and more time using the same little scripts in more files.
So I just decided to recycle the scripts in this "library".

## Why are you posting this?
Because I need validation and also think that someone could find this useful

## Where can I find it?
https://github.com/DeusAres/drawerFunctions

## Notes
There are bugs, for sure. Sometime I've rewritten things that didn't need a function (like the resize function when there is already a method for image objects) but I've kept anyway

Other than that, opinions and suggestions are very welcomed!

**Comment by nulano** [comment]
I've had a glance at the text related functions, and I would urge you to look at https://stackoverflow.com/a/70636273/1648883 again for a simple version of bounding box computation using modern functions from Pillow 8.0.0.

Similarly, for multiline bounding boxes you can use `ImageDraw.multiline_textbbox(...)`, and draw multiline text with `ImageDraw.multiline_text(...)`. Edit: Documentation is here: https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.multiline_textsize

**Reply by DeusAres** [reply]
Thanks for the suggestion! I had for long time this problem and didn't knew about the new functions, gonna clear it up it the next days so I can erase the horrible bounding box/multiline function I've written
